extraTutorials/simulation/basicSimulationPower.Rmd

```diff
@@ -83,21 +83,22 @@ hist(df$height)
 ```
 
 Now, let's create a *ground truth* for the simulated relationship between height and extroversion. 
+
   * Let's set the 'intercept' (alpha) for a person of average height as an extroversion score of 100.
   * Let's set the 'slope' (beta) such that for two increase in 1 inch of height corresponds with a 2-point increase in extroversion, on average
-  * Let's set the 'noise' to 20 -- extroversion is influenced by many factors other than height so we'll add normaly distributed error with a standard deviation of 20 to this simulation
+  * Let's set the 'noise' to 25 -- extroversion is influenced by many factors other than height so we'll add normaly distributed error with a standard deviation of 25 to this simulation
```

Comment:
I still think there's a way to do this where you don't have to hard code it in the plain text! but that's for later